README.md

```diff
@@ -14,6 +14,17 @@ If you're new to building and testing Kubernetes, it's probably best to read the
 
 The rest of this page has a summary of those steps tailored for testing clusters that have Windows nodes.
 
+## Running Windows E2E tests against your kubernetes/kubernetes PR
+
+To manually trigger Windows E2E tests to run against your PR add the following as comments on your PR:
+
+```
+/test pull-kubernetes-e2e-aks-engine-azure-windows
+/test pull-kubernetes-e2e-aks-engine-windows-containerd
+```
+
+A full list of Windows PR jobs found at https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml
```

Comment:
```suggestion
A full list of Windows PR jobs can be found at https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.jobs
```

Reply:
Super nit but other than that lgtm

Reply:
I think your suggestion breaks the link. I'll add in the missing 'can be' tho :)

Reply:
serves me right for doing it on my phone 😂